CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v1.0.0 - 2021-XX-XX
+
+### Added
+
+- Updated GoldenConfigSettings form to have a custom clean method to enforce
+a template path in the case where more than 1 repositories are selected for backup or intended repos.
+- Updated backup job to execute against multiple repos if available based on pattern matching
+- Updated intended job to execute against multiple repos if available based on pattern matching
+- Updated compliance job to execute from multiple repos if available based on pattern matching
+- Added utility function to generate the root path based on available template path
```

Comment:
The root path is a not a well understood concept, perhaps introduce that first before referring to it?